app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SessionsController < ApplicationController
+    def login_receive
+        @user = User.new
+    end
```

Comment:
`login_receive` is an unintuitive name for the controller method that will show the login page.